bindings/core/src/method_handler/call_method.rs

```diff
@@ -58,7 +58,7 @@ pub async fn call_client_method(client: &Client, method: ClientMethod) -> Respon
 }
 
 /// Call a wallet method.
-pub async fn call_wallet_method(wallet: &Wallet, method: WalletMethod) -> Response {
+pub async fn call_wallet_method(wallet: &mut Wallet, method: WalletMethod) -> Response {
```

Comment:
Requiring it mutable looks annoying

Reply:
It's actually pretty bad because it blocks everything in the bindings

Reply:
Just read the PR description, don't we only need mutability for the alias, why not just add a lock for it?